src/witness/yamlWitness.ml

```diff
@@ -233,6 +261,25 @@ struct
       ) nh []
     in
 
+    (* Generate flow-insensitive invariants *)
+    let entries = GHT.fold (fun g v acc ->
+        match g with
+        | `Left g -> (* Spec global *)
+          begin match Query.ask_global gh (InvariantGlobal (Obj.repr g)) with
+            | `Lifted inv ->
+              let invs = WitnessUtil.InvariantExp.process_exp inv in
+              List.fold_left (fun acc inv ->
+                  let invariant = Entry.invariant (CilType.Exp.show inv) in
+                  let entry = Entry.flow_insensitive_invariant ~task ~invariant in
+                  entry :: acc
+                ) acc invs
+            | `Bot | `Top -> (* TODO: 0 for bot? *)
```

Comment:
It's just the way all the other invariant queries are made and must handle that case (by not emitting an invariant). I suppose it shouldn't actually happen here, because semantically it would mean `assert(0)` holds at all program points, so the entire program is dead (including the start state), which is nonsense.

Maybe it can happen for alloced memory, but then we don't want to emit such invariant anyway, so the behavior here should be right for that.

I'll remove the copied comment because it was only regarding local states, which can be bottom, if they're dead and `assert(0)` could be used to indicate that, but that's beside the point.